packages/api-axios/src/resolve-host.js

```diff
@@ -0,0 +1,24 @@
+import { getSpecificEnv } from '@availity/env-var';
+
+const CLOUD_TO_APP_DOMAINS = {
+  tst: 'test',
+  stg: 'qa',
+  qua: 'qa',
+  prd: '',
+};
+
+export default function resolveHost(host, overrideWindow) {
+  if (host) return host;
+
+  if (/.*?\.(?:aw|az|gc)[nps|]\.availity\.com$/.test(overrideWindow.location.hostname)) {
```

Comment:
Was going to say let's try  and refactor this because of the potential vulnerability, but confirmed it is no longer being used and we can remove.